tokio/tests/rt_common.rs

```diff
@@ -1021,39 +1021,45 @@ rt_test! {
     // other tasks.
     #[test]
     fn ping_pong_saturation() {
+        use std::{sync::atomic::{Ordering, AtomicBool}};
```

Comment:
```suggestion
        use std::sync::atomic::{Ordering, AtomicBool};
```

Reply:
You know, rustfmt should have caught this.

Reply:
I'm running the rustfmt but nothing shows up... Am I doing something wrong?
```
rustfmt --check --edition 2018 $(find . -name '*.rs' -print)
```

Reply:
It's probably because of the `rt_test!` macro around it.